com.unity.multiplayer.mlapi/Runtime/Core/NetworkBehaviour.cs

```diff
@@ -99,6 +99,11 @@ internal void __endSendServerRpc(NetworkSerializer serializer, uint rpcMethodId,
             {
                 rpcQueueContainer.EndAddQueueItemToFrame(serializer.Writer, RpcQueueHistoryFrame.QueueFrameType.Outbound, NetworkUpdateStage.PostLateUpdate);
             }
+
+            if (NetworkManager.__rpc_name_table.ContainsKey(rpcMethodId))
```

Comment:
I believe it's slightly more performant to do a `TryGetValue` here so that the hash lookup only happens once